### PR TITLE
here is the book challenge

### PR DESCRIPTION
**Pull request by Dave218**
couldn't finish the bonus. couldn't figure out how to do the "display"

### CODE REVIEW

File: David/book_list_class.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	
+	def remove_title(self, title):
+		"""Remove a book from the book list"""
+		for book in self.books:
+			if book.title == title: 
+				self.books.remove(book)
+		
+	def display_titles(self):
+		"""Print out all titles currently in the book list"""
+		titles = self
```
Comment (waldenwang9966) [inline David/book_list_class.py:28]: This is incorrect. Think about how you should initialize the titles.

File: David/book_list_class.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		for book in self.books:
+			titles.append(book.title)
+			titles.sort()
```
Comment (waldenwang9966) [inline David/book_list_class.py:31]: Where should you put the sort?

File: David/book_list_class.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		 
+	def is_empty(self):
+		"""Return True if the book list is empty, False if not"""
+		pass
+
```
Comment (waldenwang9966) [inline David/book_list_class.py:36]: if len() == 0: return True else return False

**Review by waldenwang9966 — CHANGES_REQUESTED** [review]

**Comment by Dave218** [comment]
got it working. Thanks!

**Review by waldenwang9966 — APPROVED** [review]

**Pull request merged**